pkg/apis/vpc/v1alpha1/ipaddressallocation_types.go

```diff
@@ -51,6 +52,9 @@ type IPAddressAllocationSpec struct {
 	// +kubebuilder:validation:XValidation:rule="self == oldSelf",message="Value is immutable"
 	// +kubebuilder:validation:Minimum:=1
 	AllocationSize int `json:"allocationSize,omitempty"`
+	// AllocationIPs specifies the Allocated IP addresses in CIDR or single IP Address format.
```

Comment:
range is also supported.

Reply:
Allocated -> allocated?

Reply:
I've tested with NSX and found the range or comma is not supported, and NSX doc also mentions `Range or comma separated ip string is not supported in VPC IP allocation` for allocation_ips